tests/nexus/test_nexus.py

```diff
@@ -53,13 +53,12 @@ def test_nexus(tmp_path):
     """
     The nexus test function
     """
-    local_dir = os.path.abspath(os.path.dirname(__file__))
-    example_data = os.path.join(local_dir, "../data/nexus/201805_WSe2_arpes.nxs")
-
+    dirpath = os.path.join(os.path.dirname(__file__), "../data/nexus")
+    example_data = os.path.join(dirpath, "201805_WSe2_arpes.nxs")
```

Comment:
Should this file then be also updated to conform to the new definitions?

Reply:
Ah no! This is fine as this is according to the NXarpes definition and not NXmpes!